schema/core/primitive/acyclic_graph/base_node.json

```diff
@@ -0,0 +1,42 @@
+{
+    "schemaId": "core-primitive-graph-base-node",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "basic node schema (directed acyclic graph)",
+    "type": "object",
+    "properties": {
+        "flowchartId": {
+            "description": "Unique flowchart ID of node",
+            "type": "string"
+        },
+        "next": {
+            "description": "Flowchart ID of next node",
+            "type": "string"
+        },
+        "head": {
+            "description": "Whether node is head node or not",
+            "type": "boolean"
+        }
+    },
+    "required": [
+        "flowchartId"
+    ],
+    "definitions": {
+        "string-or-array-of-strings": {
```

Comment:
1. Not sure where this is used
2. If we do need it, maybe put in a definitions file at the `primitive` level?

Reply:
This was written to support directed acyclic graphs, such that the `next` field either contains one node ID or a list of node IDs. To my knowledge, this definition isn't currently used. Ok, I will move it to `primitive` then.